mesonbuild/scripts/gtkdochelper.py

```diff
@@ -71,7 +71,7 @@ def gtkdoc_run_check(cmd, cwd, library_paths=None):
             err_msg.append(out)
         raise MesonException('\n'.join(err_msg))
     elif out:
-        print(out)
+        print(out.encode(sys.stdout.encoding, 'replace').decode())
```

Comment:
If we start doing that, there is no end:
```
meson$ git grep 'print(' | wc -l
411
```

Better mass replace all print() calls with a meson wrapper function that does that, like `mlog.log()` already does. OTOH, we are in 2019 now, why do you have an env that doesn't support unicode?

Reply:
the env is the rhel7/centos7 docker, so sadly something quite common.. I used a different approach (I made sys.stdout always be UTF-8).

Reply:
> we are in 2019 now, why do you have an env that doesn't support unicode?

Because everything is terrible. For example the default build environment for Debian package building is ASCII. They are working on fixing that, though.